misc/ballerinalang-data-mapper/src/main/java/org/ballerinalang/datamapper/AIDataMapperCodeActionUtil.java

```diff
@@ -60,6 +64,12 @@
     private static Cache<Integer, String> mappingCache =
             CacheBuilder.newBuilder().maximumSize(MAXIMUM_CACHE_SIZE).build();
 
+    private static final String SCHEMA_NAME_PROPERTY = "schema";
+    private static final String FIELD_ID_PROPERTY = "id";
+    private static final String FIELD_TYPE_PROPERTY = "type";
```

Comment:
Shall we directly use `TYPE`, `ID`, `SCHEMA` as the constant names? It seems the other two words is not adding any value here

Reply:
Changed the constant names.